src/main/kotlin/lotto/domain/WinningCombination.kt

```diff
@@ -0,0 +1,20 @@
+package lotto.domain
+
+import lotto.exceptions.LottoException
+import lotto.util.LottoUtils
+
+/**
+ * the class contain winning ticket and bonus number to eval user's Lotto tickets
+ */
+class WinningCombination(val winningTicket: Lotto, val bonusNumber: Int = 0) {
+    init {
+        LottoUtils.requireOrThrow(
+            LottoUtils.isInRange(bonusNumber),
+            LottoException.InvalidBonusNumberException(bonusNumber),
+        )
+        LottoUtils.requireOrThrow(
+            winningTicket.numbers.none { it == bonusNumber },
+            LottoException.InvalidBonusNumberException(bonusNumber),
+        )
+    }
+}
```

Comment:
Nice job using `Lotto` in `WinningCombination` 👍 
By doing this, `WinningCombination` gets all the `Lotto` requirements as well!